staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -621,10 +619,8 @@ func (s *store) GetList(ctx context.Context, key string, opts storage.ListOption
 	// set the appropriate clientv3 options to filter the returned data set
 	var limitOption *clientv3.OpOption
 	limit := opts.Predicate.Limit
-	var paging bool
```

Comment:
this bool still looks relevant below because it was only true when `opts.Predicate.Limit > 0`

Reply:
done